etc/install_deps.sh

```diff
@@ -1,19 +1,19 @@
 #!/bin/bash
```

Comment:
you can copy the contents of this file into `travis_install_deps.sh` and restore the previous `install_deps.sh`. The existing `install_deps.sh` is useful for installing on client computers where as `travis_install_deps.sh` helps install the missing dependencies in travis environment.